DIS.html

```diff
@@ -68,32 +61,22 @@ <h3>Section Two - Re-Import the Game</h3>
         We will now Re-Import the game to your 3DS's System Memory.
     </p>
     <ol>
-        <li>Power off your 3DS if you haven't already</li>
-        <li>Put your SD Card into your PC</li>
-        <li>Open the SD Card's Contents</li>
-        <li>Open the Folder Named <code>Nintendo 3DS</code></li>
-        <li>Open the Folder Named like your ID0</li>
-        <li>Open its sub-folder, it has a 32-character-long name again</li>
-        <li>Open the Folder Named <code>Nintendo DSiWare</code></li>
+        <li>Power off your system and insert your SD card into your PC</li>
+        <li>Navigate to Nintendo 3ds > [Your ID0] > 32 character folder > Nintendo dsiware</code></li>

```

Comment:
Use <code> tags for paths and such to stay consistent;
Use a proper path - / instead of >

Reply:
(Stupid autoformat. Use code-tags is what it should say.)

Reply:
we tried it and it just ends up looking confusing and cluttered than ">"

Reply:
<code>SD Card/Nintendo 3DS/YOUR ID0/ 32-character-folder/NintendoDSiWare</code> ? i dont think that's cluttered?